client/src/tables/orgEmployeeGender.js

```diff
@@ -27,13 +27,13 @@ export default {
   },
 
   name: {
-    en: "Population by Employee Gender",
-    fr: "Population selon le sexe",
+    en: "Employee Gender",
+    fr: "Sexe",
```

Comment:
Double check french with someone, should it also mention "employee"? 

English should be "sex" as well, in my opinion, but we don't have to change that immediately.